README.md

```diff
@@ -62,6 +62,12 @@ brew install git-lfs
 git lfs install
 ```
 
+## Deployments
+
+This API is deployed to AWS using Docker and GitHub Actions. The API can be accessed at the following [url](http://3.254.180.26).
```

Comment:
I checked the URL and got "Not Found" - Is this intentional?

Reply:
Yep that's intentional, if you visit `http://3.254.180.26/admin/` you should get to a login page for an admin panel but you won't be able to login since we have no users in the database